### PR TITLE
A test PR.

### PR DESCRIPTION
**Pull request by ocelotpotpie**
You need to select a pull request template!

If you're adding a new feature, copy and paste the following parameter at the end of the URL:
    ?template=new-feature.md

If you're fixing a bug, copy and paste the following parameter at the end of the URL:
    ?template=bug-fix.md

For more information about contributing to EssentialsX, see CONTRIBUTING.md:
    https://github.com/EssentialsX/Essentials/blob/2.x/CONTRIBUTING.md

### CODE REVIEW

**Comment by JRoy** [comment]
wqer

**Review by JRoy — APPROVED** [review]

**Review by JRoy — CHANGES_REQUESTED** [review]
wer

**Review by JRoy — COMMENTED** [review]
zdgf

File: .github/workflows/build-master.yml [hunk 1]
```diff
@@ -36,31 +37,3 @@ jobs:
...
-
-    - name: Prepare Javadocs
-      if: ${{ success() && github.event_name == 'push' && github.repository == 'EssentialsX/Essentials' && github.ref == 'refs/heads/2.x' }}
-      run: |
-        mv Essentials/build/docs/javadoc/ javadocs/
-        cp -r EssentialsAntiBuild/build/docs/javadoc/ javadocs/EssentialsAntiBuild/
-        cp -r EssentialsChat/build/docs/javadoc/ javadocs/EssentialsChat/
-        cp -r EssentialsGeoIP/build/docs/javadoc/ javadocs/EssentialsGeoIP/
-        cp -r EssentialsProtect/build/docs/javadoc/ javadocs/EssentialsProtect/
-        cp -r EssentialsSpawn/build/docs/javadoc/ javadocs/EssentialsSpawn/
```
Comment (JRoy) [inline .github/workflows/build-master.yml:56]: fasd

File: .github/workflows/build-master.yml [hunk 2]
```diff
@@ -36,31 +37,3 @@ jobs:
...
-        cp -r EssentialsXMPP/build/docs/javadoc/ javadocs/EssentialsXMPP/
-
-    - name: Deploy Javadocs
-      if: ${{ success() && github.event_name == 'push' && github.repository == 'EssentialsX/Essentials' && github.ref == 'refs/heads/2.x' }}
-      uses: netlify/actions/cli@master
-      with:
-        args: deploy --dir=javadocs/ --prod --message="GitHubActionsDeploy"
```
Comment (JRoy) [inline .github/workflows/build-master.yml:63]: dgfg

File: .github/workflows/build-master.yml [hunk 3]
```diff
@@ -36,31 +37,3 @@ jobs:
...
-        mv Essentials/build/docs/javadoc/ javadocs/
-        cp -r EssentialsAntiBuild/build/docs/javadoc/ javadocs/EssentialsAntiBuild/
-        cp -r EssentialsChat/build/docs/javadoc/ javadocs/EssentialsChat/
-        cp -r EssentialsGeoIP/build/docs/javadoc/ javadocs/EssentialsGeoIP/
-        cp -r EssentialsProtect/build/docs/javadoc/ javadocs/EssentialsProtect/
-        cp -r EssentialsSpawn/build/docs/javadoc/ javadocs/EssentialsSpawn/
-        cp -r EssentialsXMPP/build/docs/javadoc/ javadocs/EssentialsXMPP/
-
-    - name: Deploy Javadocs
-      if: ${{ success() && github.event_name == 'push' && github.repository == 'EssentialsX/Essentials' && github.ref == 'refs/heads/2.x' }}
```
Comment (JRoy) [inline .github/workflows/build-master.yml:60]: erg

**Review by JRoy — CHANGES_REQUESTED** [review]

File: .github/workflows/build-master.yml [hunk 4]
```diff
@@ -36,31 +37,3 @@ jobs:
...
-      env:
-        ORG_GRADLE_PROJECT_essxUsername: ${{ secrets.ESSENTIALSX_DEPLOY_USERNAME }}
-        ORG_GRADLE_PROJECT_essxPassword: ${{ secrets.ESSENTIALSX_DEPLOY_PASSWORD }}
-      run: |
-        ./gradlew publish
-
-    - name: Prepare Javadocs
-      if: ${{ success() && github.event_name == 'push' && github.repository == 'EssentialsX/Essentials' && github.ref == 'refs/heads/2.x' }}
-      run: |
```
Comment (JRoy) [inline .github/workflows/build-master.yml:50]: g

File: .github/workflows/build-master.yml [hunk 5]
```diff
@@ -36,31 +37,3 @@ jobs:
       with:
         name: EssentialsX plugin jars
         path: jars/
-
-    - name: Deploy to Maven repo
-      if: ${{ success() && github.event_name == 'push' && github.repository == 'EssentialsX/Essentials' && github.ref == 'refs/heads/2.x' }}
-      env:
```
Comment (JRoy) [inline .github/workflows/build-master.yml:42]: rr

File: .github/workflows/build-master.yml [hunk 6]
```diff
@@ -36,31 +37,3 @@ jobs:
...
-        ORG_GRADLE_PROJECT_essxUsername: ${{ secrets.ESSENTIALSX_DEPLOY_USERNAME }}
-        ORG_GRADLE_PROJECT_essxPassword: ${{ secrets.ESSENTIALSX_DEPLOY_PASSWORD }}
-      run: |
-        ./gradlew publish
-
-    - name: Prepare Javadocs
-      if: ${{ success() && github.event_name == 'push' && github.repository == 'EssentialsX/Essentials' && github.ref == 'refs/heads/2.x' }}
```
Comment (JRoy) [inline .github/workflows/build-master.yml:49]: r

**Review by JRoy — APPROVED** [review]

**Comment by Draycia** [comment]
brrrr

**Pull request closed without merging**